symphony-bdk-http/symphony-bdk-http-webclient/src/main/java/com/symphony/bdk/http/webclient/ApiClientWebClient.java

```diff
@@ -246,6 +249,11 @@ private void serializeMultiPartDataEntry(String paramKey, Object paramValue,
     }
   }
 
+  private void addFileToFormParam(String paramKey, File paramValue, MultiValueMap<String, Object> formValueMap) {
+    File file = paramValue;
+    formValueMap.add(paramKey, new FileSystemResource(file));
```

Comment:
does this file get stored in the machine?

Reply:
The file has to be stored on the machine and accessible by the Java process running this. 
I guess your question is: does the file get closed after HTTP request is made. I have to check that.